manual/manual/tutorials/coreexamples.etex

```diff
@@ -677,6 +677,94 @@ print_expr e; print_newline ();;
 print_expr (deriv e "x"); print_newline ();;
 \end{caml_example}
 
+\section{Printf formats}
+
+There is a "printf" function in
+\href{https://caml.inria.fr/pub/docs/manual-ocaml/libref/Printf.html}{Printf}
+module (see chapter~\ref{c:moduleexamples}) that allows you to make formatted
+output more concisely.
+It follows the behavior of the "printf" function from the C standard library.
+The "printf" function takes a format string that describes the desired output
```

Comment:
Would `takes as argument`  be clearer?

Reply:
I don't think so. This is a fairly well-established expression, hardly anyone would think that the function takes it somehow differently.

Reply:
Ok.